Source/MultipartFormData.swift

```diff
@@ -510,14 +494,10 @@ open class MultipartFormData {
                 }
 
                 try write(&buffer, to: outputStream)
-            } else if bytesRead < 0 {
-                throw AFError.inputStreamReadFailed(inputStream: inputStream)
```

Comment:
It's documented to be so. From the `InputStream` docs:
```
Return Value
A number indicating the outcome of the operation:

A positive number indicates the number of bytes read.

0 indicates that the end of the buffer was reached.

-1 means that the operation failed; more information about the error can be obtained with streamError.
```
If we want to keep this check, we'll need another error reason, as we'll catch any error as it is.